### PR TITLE
[TR] Merge 1.4 1.5

### PR DESCRIPTION
**Pull request by willy-ahva**
<3 Thanks for taking the time to contribute! You're awesome! <3

If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/CONTRIBUTING.md 

**Description (for Contributor and Core Developer)**

What does this Pull Request do? reference the related issue?

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |
| Added Behats                      |
| Changelog updated                 |
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

File: src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -99,12 +99,10 @@ flash:
     created: "L'export a été correctement crée"
     removed: "Le profil d'export a été supprimé"
     updated: "L'export a correctement été mis à jour"
-    running: "L'export est en cours d'exécution"
```
Comment (solivier) [inline src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml]: is it normal we delete translations here ?

Comment (willy-ahva) [inline src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml]: no pb, it'll be update be crowdin

**Comment by juliensnz** [comment]
GTM

File: src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml (hunk 1 above)
Comment (solivier) [inline src/Pim/Bundle/ImportExportBundle/Resources/translations/messages.fr.yml]: I though we were not supposed to delete translations ?

**Pull request merged**